libr/crypto/p/crypto_punycode.c

```diff
@@ -0,0 +1,52 @@
+#include <r_util.h>
+#include <r_lib.h>
+#include <r_crypto.h>
+
+static int flag = 0;
+
+static int punycode_set_key(RCrypto *cry, const ut8 *key, int keylen, int mode, int direction) {
```

Comment:
yeah i didnt complained about this
> On 05 May 2016, at 14:51, Rakholiya Jenish <notifications@github.com> wrote:
> 
> In libr/crypto/p/crypto_punycode.c <https://github.com/radare/radare2/pull/4785#discussion_r62182020>:
> 
> > @@ -0,0 +1,52 @@
> > +#include <r_util.h>
> > +#include <r_lib.h>
> > +#include <r_crypto.h>
> > +
> > +static int flag = 0;
> > +
> > +static int punycode_set_key(RCrypto *cry, const ut8 *key, int keylen, int mode, int direction) {
> that's the standard signature being used in the RCryptoPlugin
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/radare/radare2/pull/4785/files/4aef3b86adc714b629693c3a1dc0140cd21ff041#r62182020>